shell/platform/embedder/embedder.h

```diff
@@ -943,6 +950,13 @@ typedef struct {
   VoidCallback destruction_callback;
 } FlutterSoftwareBackingStore;
 
+typedef struct {
```

Comment:
might be useful to have a `struct_size` parameter here, incase we decide to add any other members outside the union.